### ISSUE TITLE
Cannot install biber (or any package) on Ubuntu

### ISSUE THREAD

**Issue by yordiak** [MiKTeX/miktex-packaging#2]
Hello, 
I decided to give a try on MikTex's  ubuntu version (Ubuntu 16.04 64bit). However, I experience the following problem: biber program is missing!
 
I tried to install it using  `sudo mpm --admin --install=biber` command, but I get the following error:

> Sorry, but "MiKTeX Package Manager" did not succeed.
> The log file hopefully contains the information to get MiKTeX going again:
> /var/log/miktex/mpmcli_admin.log
> You may want to visit the MiKTeX project page, if you need help.

How can I overcome this problem?

PS: A relative problem is that I cannot launch Miktex package manager (mpm) as well. The command `sudo mpm --admin` generates an error:

> Sorry, but "MiKTeX Package Manager" did not succeed for the following reason:
> Could not restart in windowed mode.
> The log file hopefully contains the information to get MiKTeX going again:
>  /var/log/miktex/mpmcli_admin.log
> You may want to visit the MiKTeX project page, if you need help.

**Comment by edocevoli** [comment]
This is not a packaging issue.

**Comment by yordiak** [comment]
the command

`sudo mpm --admin --install biber` also produces the above error

**Comment by edocevoli** [comment]
Yes, that's because the package is not yet available for Ubuntu (see the title of this issue).

**Comment by andrewdbate** [comment]
MikTeX on Ubuntu is of no use to me without biber. Is there any reason for it not being included? Or an ETA for when it will be included?

**Comment by thymaro** [comment]
I didn't realize this, because I work on three machines on the same file. Anyways, my workflow for biber now looks like:

  1. compile on windows, which produces a .bbl
  1. don't delete .bbl
  1. continue work on ubuntu
  1. repeat when new references are added to the document

fortunately, latex will run whether biber is available or not, the document will just have the bibkeys instead of the formatted references. Works for me atm, but still, an ETA would be appreciated.

**Comment by andreipoe** [comment]
You can always instal biber manually from [its homepage](http://biblatex-biber.sourceforge.net/) until it's included in MikTeX.

**Comment by niklasholm** [comment]
This should be marked as critical. A LaTeX distro without biber is useless.

**Comment by moewew** [comment]
This came up again in https://tex.stackexchange.com/q/471704/35864. Is there anything from the Biber side that would have to be done to get Linux and Mac Biber into MikTeX? Naively I would have assumed it would be enough to grab the binaries from CTAN https://www.ctan.org/tex-archive/biblio/biber/binaries and pack them up, much like what I assume is happening for the Windows binaries at the moment.

**Comment by edocevoli** [comment]
Implementing this feature requires some MiKTeX source code modifications. I will move this ticket into the other repository.
